db/migrate/20211104183055_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[6.1]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.text :photo
```

Comment:
* Kindly change `t.text: photo` to `t.string: photo` as the photo will be a link to an image so therefore the right type would be `string`